addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2159,10 +2159,36 @@ const Wysiwyg = Widget.extend({
         $('.o_editable')
             .removeClass('o_editable o_is_inline_editable o_editable_date_field_linked o_editable_date_field_format_changed');
 
-        const defs = _.map(_.groupBy($allBlocks.toArray(),
-            function($obj) {
-            return $obj.dataset['oe-model'] + $obj.dataset['oe-field'] + $obj.dataset['oe-id']
```

Comment:
@HydrionBurst This was not working at all, did not respect any linter convention and most importantly... just broke the edition of website (see commit message for details). The main problem was that `.dataset['oe-model']` is not the correct way to read the dataset and thus always returned `undefined`. And the sum you did return `NaN`... I understand we are in a rush but please let's be more careful, especially when touching tricky code of other teams. Ask us for help when necessary.

There were way more things to consider (see this PR, hopefully it restores everything).

Reply:
Indeed, I did not know about that MacOS + Safari subtility, good to know :+1: However, even without that, the website edition was still broken as view parts were regrouped and sent to `_saveElement` as one. But clearly already more understandable you did not see it because of that MacOS + Safari stuff. In any case, no worries, we all make mistakes (I should have seen it as I was code-owner on the original PR, so the mistake was mine too), just wanted to let you know that when touching the editor, be extra careful and do not hesitate to ask for help :+1: